Ports/dosbox-staging/package.sh

```diff
@@ -7,15 +7,15 @@ configopts=(
     '-Ddynamic_core=none'
     '-Dtry_static_libs=opusfile'
     '-Dunit_tests=disabled'
-    '-Duse_fluidsynth=false'
     '-Duse_mt32emu=false'
     '-Duse_opengl=false'
-    '-Duse_png=false'
+    '-Duse_png=true'
```

Comment:
This change is probably irrelevant to this commit, seeing that you even remove it in the next one?

Reply:
Ah, yes of course